### PR TITLE
Better clarification for question 27

### PR DESCRIPTION
**Pull request by wilbo**
Might be a little far-fetched, but still...

The `continue` statement always skips an iteration. It is up to the developer to let this happen in a condition.

### CODE REVIEW

**Comment by lydiahallie** [comment]
I think the explanation is quite obvoius, but you could rephrase it to: 

``` The `continue` statement skips an iteration if a certain condition returns `true`,  in this case if `i` is equal to `3`. ```

**Comment by wilbo** [comment]
> The `continue` statement skips an iteration if a certain condition returns `true`, ...

This part of the sentence is simply not true. The `continue` statement is not dependant on a (certain) condition. 

I rephrased it in a way you might like it better.

**Comment by mindsers** [comment]
I agree that this sentence should be improved. But my question is: **how enforce consistancy between translation?**

If this PR is merged, the decision should impact every translation available. How to be sure it is updated in other languages?

**Comment by lydiahallie** [comment]
I'll close this PR now due to inactivity. If it's still an issue, feel free to reopen it! 😃
